helix-term/src/commands.rs

```diff
@@ -4259,6 +4260,13 @@ fn dap_init(cx: &mut Context) {
     let request = debugger.initialize("go".to_owned());
     let _ = block_on(request).unwrap();
 
```

Comment:
I had these as one request but the DAP spec diagram seems to send breakpoints before calling launch:

> This configuration information must be passed to the debug adapter before program execution starts. Some debuggers are able to deal with this information very early, even before the debuggee is known; other debuggers accept this information only when the debuggee is about to start running.

Reply:
Execution starts after `configurationDone` request. `launch` only loads the program.

Reply:
Per spec it should be separate. emacs `dap-mode` follows this approach, `nvim-dap` matches our aproach.